Project/Source Files/Screens/Recipies/RecipiesTableViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RecipiesViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class RecipiesTableViewController: UIViewController {
+
+    @IBOutlet var tableView: UITableView!
+    var dataBase = [
```

Comment:
Here it would be also nice to add documentation stating that this is used for mockup only and soon to be removed :) 
something like
> // TODO: Remove when the API implementation will be ready.